tools-reference/sed/text.xml

```diff
@@ -65,7 +65,7 @@ constructs are specific to <c>GNU sed 4</c> <d/> on non-GNU userland archs, the
 guaranteed to be installed as part of <c>@system</c>. This was not
 always the case,
 which is why some packages, particularly those which use <c>sed -i</c>, have
-<c>DEPEND</c> s upon <c>&gt;=sys-apps/sed-4</c>.
+<c>BDEPEND</c> s upon <c>&gt;=sys-apps/sed-4</c>.
```

Comment:
What's that lonely "s" doing here?

Also, this is about old packages, which most certainly have &gt;=sed-4 in DEPEND, not in BDEPEND.

Reply:
Oh, duh. Let's fix the s though.